common/core/web/utils/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "./dist/index.js",
   "types:": "./dist/index.d.ts",
   "scripts": {
-    "build": "cd src/ && ./build.sh -skip-dependency-install",
+    "build": "./build.sh -skip-dependency-install",
```

Comment:
So now Windows will say "." is not a valid command...  because `./build.sh` should be `.\\build.sh` on Windows (and that's assuming that bash is configured to launch shell scripts from command prompts, which is environmentally dependent.

Here's how we launch bash scripts on Windows on the build agents:
```
"C:\Program Files\Git\bin\bash.exe" --init-file "c:\Program Files\Git\etc\profile" -l <script.sh>
```

So `"C:\Program Files"` will be called `"C:\Programme"` in German versions of Windows etc. So instead we can use `%ProgramFiles%` but then if the user has installed Git for Windows 32 bit instead of Git for Windows 64 bit it will be found in `%ProgramFiles(x86)%` etc. And it's also possible to install programs in custom locations, in which case they need to configure that ALL OVER AGAIN.

For the Windows builds, we have a "GIT_BASH_FOR_KEYMAN" environment variable. Maybe we just need to make that mandatory?

Here's what we do in Windows makefiles:
```
!ifdef GIT_BASH_FOR_KEYMAN
    $(GIT_BASH_FOR_KEYMAN) build.sh release/fv
    $(GIT_BASH_FOR_KEYMAN) build.sh release/packages/fv_all
!else
    start /wait ./build.sh release/fv
    start /wait ./build.sh release/packages/fv_all
!endif
```

And we define:

```
SET GIT_BASH_FOR_KEYMAN="C:\Program Files\Git\bin\bash.exe" --init-file "c:\Program Files\Git\etc\profile" -l
```

But that's still assuming Windows and won't work cross platform.

Reply:
Gosh, what a nightmare! Maybe I _will_ try `run-script-os` after all...